base/errorshow.jl

```diff
@@ -562,9 +562,9 @@ function replaceuserpath(str)
     return str
 end
 
-const STACKTRACE_MODULECOLORS = [:light_blue, :light_yellow,
-        :light_magenta, :light_green, :light_cyan, :light_red,
-        :blue, :yellow, :magenta, :green, :cyan, :red]
+const STACKTRACE_MODULECOLORS = [:magenta, :blue, :cyan, :green, :yellow]
+const STACKTRACE_FIXEDCOLORS = Dict(Base => :red, Core => :red)
```

Comment:
This doesn't seem to work, or gets the wrong version of `Base` somehow? After building this branch: 
```
julia> Base.STACKTRACE_FIXEDCOLORS
Dict{Module, Symbol} with 2 entries:
  Core => :red
  Base => :red

julia> Base.STACKTRACE_FIXEDCOLORS[Base]
ERROR: KeyError: key Base not found
Stacktrace:
 [1] getindex(h::Dict{Module, Symbol}, key::Module)
   @ Base ./dict.jl:482
 [2] top-level scope
   @ REPL[11]:1
```

Reply:
You need to rehash the Dict in `__init__` (`Base.rehash!`) or use an `IdDict`. The serializer hashes are not valid in the new process.

Reply:
Thanks, `IdDict` seems simplest, and seems to work fine.